docs/_guide/styles.md

```diff
@@ -162,327 +122,618 @@ class MyElement extends LitElement {
+
+Styles you add to a component can affect:
+
+* [The shadow tree (your component's rendered template)](#shadowroot).
+* [The component itself](#host).
+* [The component's children](#slot).
 
-```js
-import {LitElement, property} from 'lit-element';
 
+### Style the shadow tree {#shadowroot}
+
+LitElement templates are rendered into a shadow tree by default. Styles scoped to an element's shadow tree don't affect the main document. Similarly, with the exception of [inherited CSS properties](#inheritance), document-level styles don't affect the contents of a shadow tree.
+
+When you use standard CSS selectors, they only match elements in your component's shadow tree.
+
+```js
 class MyElement extends LitElement {
-  @property() mainColor = 'blue';
+  static get styles() {
+    // Write styles in standard CSS
+    return css`
+      * { color: red; }
+      p { font-family: sans-serif; }
+      .myclass { margin: 100px; }
+      #main { padding: 30px; }
+      h1 { font-size: 4em; }
+    `;
+  }
   render() {
     return html`
-      <style>
-        :host {
-          color: ${this.mainColor};
-        }
-      </style>
+      <p>Hello World</p>
+      <p class="myclass">Hello World</p>
+      <p id="main">Hello World</p>
+      <h1>Hello World</h1>
     `;
   }
 }
 ```
 
-### Define styles in an external stylesheet {#external-stylesheet}
+{% include project.html folder="style/styleatemplate" openFile="my-element.js" %}
+
+### Style the component itself
+
+Styles defined in an element's shadow tree can style the element itself (also known as the _shadow host_).
 
-We strongly recommend static styles, CSS custom properties, or [lit-html's `classMap` or `styleMap` directives](TODO) if you're styling non-host shadow root contents.
+To create default styles for the host element, use the `:host` CSS pseudo-class and `::host()` CSS pseudo-class function.
 
-However, you can load an external stylesheet into a shadow root with a `<link>` element:
+*   `:host` selects the host element.
+
+*   <code>:host(<var>selector</var>)</code> selects the host element, but only if the host element matches _selector_.
 
 ```js
-import {LitElement} from 'lit-element';
+static get styles() {
+  return css`
+    /* Selects the host element */
+    :host { 
+      display: block; 
+    }
+
+    /* Selects the host element if it is hidden */
+    :host([hidden]) { 
+      display: none; 
+    }
+
+    /* Selects the host element if it has class "blue" */
+    :host(.blue) { 
+      background-color: aliceblue;
+      color: blue;
+    }
+  `;
+}
+```
+
+{% include project.html folder="style/host" openFile="my-element.js" %}
+
+Note that the host element can be affected by styles from outside the shadow tree, as well, so you should consider 
+the styles you set in `:host` and `:host()` rules as _default styles_ that can be overridden by the user. For example:
+
+```css
+my-element {
+  display: inline-block;
+}
+```
+
+### Style the component's children
```

Comment:
This section is missing any newbie context. It's pretty close to pure jargon. 

"If your component will have children (like the built-in <select> element has <option>s and <ul> has <li>s), your template must contain placeholders for where the children should be placed inside the rendered template. These are called slots, and are specified with the <slot> element:"

Reply:
Yep, we can do better here. I'll take a stab.

Reply:
Actually, there is a separate section on `<slot>` in the templates chapter. It seems like that would be a better place for more detailed background information. I might rephrase this sentence a little bit, but will point to the other section.